readme.md

```diff
@@ -1,8 +1,63 @@
+**QR - Dart** is a QR code generation library written in Dart.
+
 [![Build Status](https://travis-ci.org/kevmoo/qr.dart.svg?branch=master)](https://travis-ci.org/kevmoo/qr.dart)
 
-## Dart QR code generation library.
+# Features
+
+- Supports QR code versions 1 - 40
+- Error correction / redundancy
+
+# Installing
+
+You can install the package by adding the following lines to your `pubspec.yaml`:
+
+```yaml
+dependencies:
+  qr: "^0.1.0+1"
+```
+
+After adding the dependency to your `pubspec.yaml` you can run: `pub get` or `flutter packages get` if you're using Flutter.
+
+# Getting started
+
+To start, import the dependency in your code:
+
+```dart
+import 'package:qr/qr.dart';
+```
+
+To build your QR code data you should do so as such:
+
+```dart
+final _qrCode = new QrCode(4, QrErrorCorrectLevel.L);
```

Comment:
Sorry for being picky. You create this like a private var – but then use it likes it's in imperative code.

Should drop the `_`

Reply:
No worries. Yeah, I had it set up different in my project so I think I was on autopilot but this makes sense.